draft-ietf-mops-streaming-opcons.md

```diff
@@ -751,6 +757,32 @@ For most of the history of the Internet, these transport protocols, described in
 
 For this reason, we have included a description of how the path characteristics that streaming media providers may see are likely to evolve over time.
 
+## Transport Protocols and Media Transport Protocols {#mtp}
+
+Within {{sec-trans}}, the term "Media Transport Protocol" is used to describe to describe the protocol of interest. This is easier to understand if the reader assumes a protocol stack that looks something like this:
+
+~~~~
+               Media
+    ---------------------------
+           Media Format
+    ---------------------------
+    Media Transport Protocol(s)
+    ---------------------------
+      Transport Protocol(s)
+~~~~
+
+where
+
+* "Media Format" would be something like an RTP payload format {{RFC2736}} or ISOBMFF {{ISOBMFF}},
+* "Media Transport Protocol" would be something like RTP or HTTP, and
+* "Transport Protocol" would be something like TCP or UDP.
+
+Not all possible streaming media applications follow this model, but for the ones that do, it seems useful to have names for "the protocol layers beteern Media and the Transport Layer".
```

Comment:
s/names for "the protocol layers beteern Media and the Transport Layer"/names for the protocol layers between Media Format and Transport Layer
(typo and removal of the quotation)

Reply:
Yes, and thank you for the suggestion!